xapian-letor/configure.ac

```diff
@@ -34,10 +34,30 @@ dnl 0:0:0 1.3.1 First release with xapian-letor split out from xapian-core
 LIBRARY_VERSION_INFO=0:0:0
 AC_SUBST(LIBRARY_VERSION_INFO)
 
+LIBRARY_VERSION_SUFFIX=
+dnl Where xapian.h, etc go.
```

Comment:
`xapian.h` is a header from xapian-core...

Reply:
Also this comment was copied from core, but the line of code it refers to wasn't (it's currently handled further down in letor's `configure.ac` for some reason) so it doesn't actually make sense as the code here currently is.